common/honeycomb/honeycomb.go

```diff
@@ -1,13 +1,28 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
the linting on the project required these, which is why i added them. will probably be required for an upstream merge